src/wast-lexer.cc

```diff
@@ -129,10 +193,15 @@ std::unique_ptr<WastLexer> WastLexer::CreateBufferLexer(const char* filename,
   return std::unique_ptr<WastLexer>(new WastLexer(std::move(source), filename));
 }
 
+bool WastLexer::lookahead_is_lpar() {
```

Comment:
nit: `IsLookaheadLpar`. I've been trying to use CamelCase for new functions (gonna switch it all over at some point...)